src/function/pragma/pragma_functions.cpp

```diff
@@ -43,6 +53,16 @@ void RegisterEnableProfiling(BuiltinFunctions &set) {
 	set.AddFunction("enable_profiling", functions);
 }
 
+void RegisterDetailedEnableProfiling(BuiltinFunctions &set) {
+	vector<PragmaFunction> functions;
+	functions.push_back(PragmaFunction::PragmaStatement(string(), PragmaEnableDetailedProfilingStatement));
+	functions.push_back(
+	    PragmaFunction::PragmaAssignment(string(), PragmaEnableDetailedProfilingAssignment, LogicalType::VARCHAR));
+
+	set.AddFunction("enable_detailed_profile", functions);
```

Comment:
I think it would be more clear to the user to turn this into a toggle that specifies what kind of profiling is displayed, e.g.:

```
PRAGMA profiling_mode ='standard'
PRAGMA profiling_mode='detailed'
```